### PR TITLE
feat(linker): consolidator user testing feedback

### PR DESCRIPTION
**Pull request by HejdaJakub**
* Forkflow changed - when user tries to send message to user support, new dialog will be opened. In
this dialog is textarea with general message which can be changed by the user. This message will be merged with some technical info before sending to user support.
* Some texts were too.

### CODE REVIEW

File: apps/linker/src/app/components/send-message-dialog/send-message-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<div>
+  <div mat-dialog-title>
+    <h2 class="page-subtitle">
```
Comment (xkureck) [inline apps/linker/src/app/components/send-message-dialog/send-message-dialog.component.html:3]: add css style for the page-subtitle class - in linker it is not defined

File: libs/lib-linker/src/lib/consolidation-result/consolidation-result.component.html [hunk 2]
```diff
@@ -12,9 +12,16 @@
       <h1>
         {{('SHARED_LIB.CONSOLIDATOR.' + result + '_TITLE') | translate}}
       </h1>
-      <h5>
+      <h5 *ngIf="!(userMail && result === 'OK')">
         {{('SHARED_LIB.CONSOLIDATOR.' + result + '_SUBTITLE') | translate}}
-        <span *ngIf="result === 'UNKNOWN_ERROR'"> {{supportMail}} </span>
+        <span *ngIf="result === 'UNKNOWN_ERROR' || result === 'MESSAGE_SENT_TO_SUPPORT'">
+          {{supportMail}}
+        </span>
```
Comment (xkureck) [inline libs/lib-linker/src/lib/consolidation-result/consolidation-result.component.html:19]: oh sorry, you misunderstood me
I wanted to show the user to what HIS email will support contact him - so not the perun@cesnet.cz but -> eg. luckureckova@gmail.com

**Review by xkureck — CHANGES_REQUESTED** [review]
image

File: apps/linker/src/app/components/send-message-dialog/send-message-dialog.component.html (hunk 1 above)
Comment (HejdaJakub) [reply apps/linker/src/app/components/send-message-dialog/send-message-dialog.component.html]: Css class added.

File: libs/lib-linker/src/lib/consolidation-result/consolidation-result.component.html (hunk 2 above)
Comment (HejdaJakub) [reply libs/lib-linker/src/lib/consolidation-result/consolidation-result.component.html]: Oh my bad, I changed the email and text for this use case (MESSAGE_SENT_TO_SUPPORT).

**Review by xkureck — APPROVED** [review]

**Pull request merged**